### DISCUSSION TITLE
跨分区rollup是怎么处理的？

### DISCUSSION THREAD

**Discussion by boluochuile** [apache/doris#7305] [RFC]
base表中有年、月、周和日四种日期维度，分区键为dt（日）。
现在为了满足周维度的数据聚合创建了一个聚合到周粒度的rollup，也即是rollup表中不存在分区键dt了，只剩周、月、年三种时间维度，
那么对于这种跨分区聚合的rollup在Doris里面是怎么实现的呢？这种rollup还有分区键吗？

本人目前看了一些资料，感觉这种rollup底层其实还是以dt进行分区存储的？并不能按周粒度聚合后的数据进行存储？

**Comment by morningman** [comment]
In your case, currently Doris can not support rollup to month and year. Because the rollup table is partitioned by day.
